src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -80,6 +80,11 @@ public enum LanguageVersion
         CSharp7_2 = 702,
 
         /// <summary>
+        /// C# language version 7.3
+        /// </summary>
+        CSharp7_3 = 703,
```

Comment:
:memo: Filed https://github.com/dotnet/roslyn/issues/22957 to remind to add the drop-down entry (in project-system).